marine_sensor_msgs/msg/RadarSector.msg

```diff
@@ -0,0 +1,37 @@
+# A collection of marine radar rays, typically covering a contiguous
+# arc of a full circle.
+
+# timestamp in the header is the acquisition time of
+# the first ray in the sector.
+#
+# In frame frame_id, angles are measured around
```

Comment:
Do we have a convention about Z being up or down for these sensors? 

Is "forward" defined by the sensor itself (e.g. sonars have a definite "center of beam pattern"), or is it pretty arbitrary how they're mounted?

Reply:
Marine radars are usually mounted so the beams are parallel with the sea surface. The antenna mechanically rotates 360 degrees so a single beam is used for full coverage. 

Unlike multibeam sonars which can be mounted in various orientations for different purposes, I'm not aware of marine radars being mounted in other orientations. They do have a defined forward orientation and if they get mounted with forward not pointing towards the bow of the vessel, they typically have a correction parameter to compensate. On a crewed vessel, the radar display is often overlayed on a chart so it's important to have the data represent the correct orientation so the display is correct.

I used REP103's recomended axis orientation related to a body frame. On a typical instalation, the radar axis orientation match the base_frame orientation. Is there a different REP I'm missing that might have recommended orientations for different types of sensors?

Reply:
REP 103 conflicts with longstanding convention in the marine domain, which is x forward, y right, z down (or NED) ... so I always want to double-check which one is being used. 

I just went to look at our frames document, and realized that it shows `base_link` as in REP103; I didn't expect this, but that's fine.